backend/graminsta/user/models.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: UTF-8 -*-
+
+"""models.py
+"""
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class UserInfo(models.Model):
+    """UserInfo Class
+    """
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    age = models.IntegerField()
+    gender = models.CharField(max_length=10)
+    reg_time = models.DateTimeField(auto_now=True)
```

Comment:
I think Django user model already have this field.

See also: https://docs.djangoproject.com/en/3.0/ref/contrib/auth/#django.contrib.auth.models.User.date_joined